documentation/manual/working/javaGuide/main/akka/code/javaguide/akka/async/Application.java

```diff
@@ -15,7 +15,7 @@
     return CompletableFuture.supplyAsync(this::longComputation)
         .thenApply((Integer i) -> ok("Got " + i));
   }
-  // ###skip: 1
+
```

Comment:
We don't want the fake `longComputation` to be shown, instead we should hide it completely, by skipping all of its lines:
```suggestion
  // ###skip: 3
```